src/js/components/picklist/picklist.scss

```diff
@@ -67,8 +67,18 @@
   font-size: 14px;
   font-style: italic;
   padding: 4px 16px;
+}
 
-  .icon:first-child {
```

Comment:
I couldn't find anywhere where this is used. No results text that has an icon as well.